frontend/src/metabase/containers/ItemPicker/ItemPickerView.tsx

```diff
@@ -59,6 +62,7 @@ function ItemPickerView({
   searchString,
   searchQuery,
   showSearch = true,
+  useDebouncedSearch = false,
```

Comment:
I believe we don't need a separate prop here since we don't want to have one `ItemPicker` with search that supports typeahead and another one that doesn't. Could you rely on `isSearchEnabled` instead?

Reply:
Yeah that was the main question I had about the behavior. Would we be ok with a typeahead for all components that implement `ItemPicker` then?

Reply:
Then we definitely should be relying on `isSearchEnabled`